src/stores/WidgetsStore.ts

```diff
@@ -140,7 +152,22 @@ export class WidgetsStore {
         }
     };
 
-    // region Golden Layout Widgets
+    // create drag source for ToolbarMenuComponent
```

Comment:
can this be made `private static`?

Reply:
cant be static because the need of appStore instance for props in glConfig. is there a better way for this?

Reply:
ah how about I add a new parameter of appStore

Reply:
modification added